forward_engineering/helpers/alterScriptHelpers/alterColumnHelpers.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = (app, ddlProvider) => {
+	const _ = app.require('lodash');
+	const { checkFieldPropertiesChanged, modifyGroupItems, setIndexKeys } = require('./common')(app);
+	const { createColumnDefinitionBySchema } = require('./createColumnDefinition')(_);
+	const { AlterScriptDto } = require('./types/AlterScriptDto');
+
+	const getRenameColumnScripts = (collectionProperties, fullName) => {
```

Comment:
Move getRenameColumnScripts & getChangeTypeScripts to own files (into alterColumnHelpers folder). Use PostgreSQL for reference